src/core/types/Vault.ts

```diff
@@ -35,6 +35,8 @@ export interface GeneralVaultView {
   pricePerShare: string;
   allowZapIn: boolean;
   allowZapOut: boolean;
+  zapInWith?: string;
+  zapOutWith?: string;
```

Comment:
Should we add a TODO here as well? Maybe worth to add the same type I've mentioned here https://github.com/yearn/yearn-finance-v3/pull/674/files#r875332401 for now just in case

Reply:
yep, changed here https://github.com/yearn/yearn-finance-v3/pull/674/commits/b70f2e7d0cd1238378dc4a49fa0fdee00c0bd66b and here https://github.com/yearn/yearn-finance-v3/commit/7170e91d3685d21e399b3e874c82d785876763dd as suggested.